tests/acceptance/test_acceptance/test_decide.py

```diff
@@ -78,6 +110,53 @@ def test_decide__feature(session_obj, flag_key, expected_response, expected_stat
     resp.raise_for_status()
 
 
+@pytest.mark.parametrize(
+    "flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation", [
+        ("feature_2", expected_forced_decision_without_rule_key, 200, "feature_2", None, "variation_1"),
+        ("feature_2", expected_forced_decision_with_rule_key, 200, "feature_2", "feature_2_test", "variation_2")
+    ],
+    ids=["variation_1", "16931381940"])
+def test_decide_with_forced_decision__feature(session_obj, flag_key, expected_response, expected_status_code, forced_flag, forced_rule, forced_variation):
+    """
+    Test validates:
+    Correct response when valid or empty rule key is passed in forced-decision parameters.
+    ...
+    :param session_obj:
+    :param flag_key:
+    :param expected_response:
+    :param expected_status_code:
+    :param forced_flag:
+    :param forced_rule:
+    :param forced_variation:
+    """
+    rule_key = '"ruleKey": "{}",'.format(forced_rule) if forced_rule else ''
+    payload = """
+        {
+          "userId": "matjaz",
+          "decideOptions": [
+              "ENABLED_FLAGS_ONLY",
+              "INCLUDE_REASONS"
+          ],
+          "userAttributes": {"attr_1": "hola"},
+          "forcedDecisions": [
+            {
+              "flagKey": \"""" + forced_flag + """\",
+              """+ rule_key +"""
+              "variationKey": \"""" + forced_variation + """\"
+            }
```

Comment:
The use of triple quotes looks a little confusing. Is that the only way? What about single quotes?